ui/src/message_popup/hint_card.jsx

```diff
@@ -49,7 +49,7 @@ export default React.createClass({
         {hidden &&
           (<div>
             <div style={styles.buttonRow}>
-              <div style={styles.prompt}>Need a hint?</div>
+              <div></div>
```

Comment:
The blank div was there so that I could use the same style for the buttonRow with flex `justifyContent: 'space-between'` for both when the example is hidden and when it is shown.